src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -73,10 +73,11 @@ const getParcelsForDelivery = async (parcelIds: string[]): Promise<ParcelsForDel
             return { data: null, error: { type: "noCollectionCentre", logId: logId } };
         }
 
-        let labelCount = 0;
-        if (parcel.events && parcel.events.length > 0 && parcel.events[0].event_data) {
-            labelCount = Number.parseInt(parcel.events[0].event_data);
-        }
+        const lastEventDataValue =
+            parcel.events && parcel.events.length > 0 && parcel.events.slice(-1)[0].event_data
+                ? parcel.events.slice(-1)[0].event_data
+                : null;
+        const labelCount = lastEventDataValue ? Number.parseInt(lastEventDataValue) : 0;
```

Comment:
I think we can simplify this a bit:
```
const labelCount =
    parcel.events?.[parcel.events.length - 1] !== undefined
        ? parseInt(parcel.events[parcel.events.length - 1].event_data)
        : 0;
```

This whole function's a little bit janky because supabase doesn't have type safety